[2024-05-23 09:28:17,240] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: workflow.create_table scheduled__2021-03-07T00:00:00+00:00 [queued]>
[2024-05-23 09:28:17,251] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: workflow.create_table scheduled__2021-03-07T00:00:00+00:00 [queued]>
[2024-05-23 09:28:17,253] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-05-23 09:28:17,255] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2024-05-23 09:28:17,255] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-05-23 09:28:17,268] {taskinstance.py:1397} INFO - Executing <Task(MySqlOperator): create_table> on 2021-03-07 00:00:00+00:00
[2024-05-23 09:28:17,274] {standard_task_runner.py:52} INFO - Started process 320 to run task
[2024-05-23 09:28:17,280] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'workflow', 'create_table', 'scheduled__2021-03-07T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/workflow.py', '--cfg-path', '/tmp/tmpsppfc4i3', '--error-file', '/tmp/tmpqa1m975k']
[2024-05-23 09:28:17,282] {standard_task_runner.py:80} INFO - Job 3: Subtask create_table
[2024-05-23 09:28:17,377] {task_command.py:371} INFO - Running <TaskInstance: workflow.create_table scheduled__2021-03-07T00:00:00+00:00 [running]> on host 81e6d48ea239
[2024-05-23 09:28:17,488] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=workflow
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2021-03-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-07T00:00:00+00:00
[2024-05-23 09:28:17,489] {mysql.py:82} INFO - Executing: CREATE DATABASE qweql
[2024-05-23 09:28:17,503] {base.py:68} INFO - Using connection ID 'mysql_db' for task execution.
[2024-05-23 09:28:17,528] {dbapi.py:231} INFO - Running statement: CREATE DATABASE qweql, parameters: None
[2024-05-23 09:28:17,549] {dbapi.py:239} INFO - Rows affected: 1
[2024-05-23 09:28:17,584] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=workflow, task_id=create_table, execution_date=20210307T000000, start_date=20240523T092817, end_date=20240523T092817
[2024-05-23 09:28:17,616] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-05-23 09:28:17,692] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
